.github/workflows/amplify-deploy.yml

```diff
@@ -49,12 +49,31 @@ env:
   BRANCH_NAME: ${{ github.event_name == 'pull_request' && format('pr{0}', github.event.number) || (github.ref == 'refs/heads/main' && 'main' || github.ref == 'refs/heads/master' && 'master') }}
   BUCKET_NAME: ${{ github.event_name == 'pull_request' && inputs.bucket-name-preview || (github.ref == 'refs/heads/main' && inputs.bucket-name-main || github.ref == 'refs/heads/master' && inputs.bucket-name-main) }}
   TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
-  ZIP_NAME: ${{ github.event_name == 'pull_request' && format('{0}-{1}-preview.zip', inputs.package-name, github.event.number) || (github.ref == 'refs/heads/main' && format('{0}-main.zip', inputs.package-name) || github.ref == 'refs/heads/master' && format('{0}-master.zip', inputs.package-name)) }}
 
 jobs:
   deploy:
     runs-on: ${{ inputs.os }}
     steps:
+      # Process package name to remove prefix
+      - name: Process package name
```

Comment:
```suggestion
      - name: Remove scope from package name
```

Reply:
If it's ok (and to save my sanity) can I include this in the follow-up PR I'm spinning up straight after this one gets merged 😄